.prettierrc

```diff
@@ -3,5 +3,6 @@
   "printWidth": 100,
   "semi": true,
   "singleQuote": true,
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "endOfLine": "crlf"
```

Comment:
I think we should leave this as the default (`lf`). Is there a specific reason you had to change it to `crlf`?

Reply:
@koredefashokun the dev is using windows so it requires adding this to make the lint work properly. however, there is no need to commit to the PR. I already asked the dev to reset this file.